### PR TITLE
Clean-up outdated bindings in API servlet module

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Remove outdated bindings in OnPremisesApiServletModule

### What issues does this PR fix or reference?
https://github.com/codenvy/codenvy/issues/1587

#### Changelog
Cleaned up outdated API servlet module bindings.

#### Release Notes
N/A

### CODE REVIEW

**Comment by skabashnyuk** [comment]
@codenvy/pm review

File: assembly/assembly-wsmaster-war/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiServletModule.java [hunk 1]
```diff
@@ -52,31 +49,15 @@
                                                                  "/github/*",
                                                                  "/bitbucket/*",
                                                                  "/ssh-keys/*",
```
Comment (sleshchenko) [inline assembly/assembly-wsmaster-war/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiServletModule.java]: Are you sure that we need this binding?

Comment (mkuznyetsov) [inline assembly/assembly-wsmaster-war/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiServletModule.java]: Thx, I removed it

**Comment by JamesDrummond** [comment]
Removed doc section and added N/A for release notes.

**Pull request merged**